tests/ui/question_mark.stderr

```diff
@@ -0,0 +1,16 @@
+error: this block may be rewritten with the `?` operator
+ --> $DIR/question_mark.rs:2:5
+  |
+2 |     if a.is_none() {
```

Comment:
That's not the correct span. It should hilight the entire `if a.is_none() { return None; }`

Reply:
I've fixed the span. Can you check it now?

Thanks!

Reply:
Jup, this is perfect now.